src/commands.py

```diff
@@ -74,6 +74,19 @@ def difficulties(ep):
     .format(current["powMin"], match_pattern(int(current["powMin"]))[0], diffi["block"], issuers, len(diffi["levels"]),
     tabulate(sorted_diffi, headers="keys", tablefmt="orgtbl", stralign="center")))
 
+network_sort_keys = "block,member,diffi,uid".split(",")
```

Comment:
Why don't you store them on a [list like here](https://github.com/duniter/silkaj/blob/master/src/commands.py#L15).

Two lines breaks are needed to respect `pep8`: [like here](https://github.com/duniter/silkaj/blob/master/src/tools.py#L147-L150).